rules/windows/persistence_startup_folder_file_written_by_suspicious_process.toml

```
@@ -13,7 +13,49 @@ from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
-name = "Shortcut File Written or Modified for Persistence"
+name = "Startup Persistence by a Suspicious Process"
+note = """## Triage and analysis
+
+### Investigating Startup Persistence by a Suspicious Process
+
+Windows Startup folder is a special folder in Windows. Programs added to this folder are executed during the logon of an
+account, without user interaction, making it an excellent opportunity for attackers to maintain persistence.
+
+This rule will monitor for commonly abused processes writing to the startup folder locations.
+
+#### Possible investigation steps:
+
+- Investigate process execution chain (parent process tree).
+- Investigate other alerts related to the user/host in the last 48 hours.
+- Validate the activity is not related to planned patches, updates, network administrator activity or legitimate software
+installations.
+- Determine if activity is unique by validating if other machines in same organization have similar entry.
+- Retrieve the file:
+  - Use a sandboxed malware analysis system to perform analysis.
+    - Observe attempts of contacting external domains and addresses.
+  - Use the PowerShell Get-FileHash cmdlet to get the SHA-256 hash value of the file.
+    - Search for the existence of this file in resources like VirusTotal, Hybrid-Analysis, CISCO Talos, Any.run, etc.
+
+### False Positive Analysis
+
+- Administrators may add programs to this mechanism via command-line shells. Before the further investigation, it should
+be verified that this activity is not benign.
+
+### Related Rules
+
+- Suspicious Startup Shell Folder Modification
+- Persistent Scripts in the Startup Directory
+
+### Response and Remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- Isolate the involved host to prevent further post-compromise behavior.
+- If the triage identified malware, search the environment for additional compromised hosts.
+  - Implement any temporary network rules, procedures, and segmentation required to contain the malware.
+  - Immediately block the IoCs identified.
```

```suggestion
  - Immediately block external network domains and addresses communicating with malware if optional.
```

---

I think that IoCs can be a little more broad and include other artifacts found in the triage and response, maybe having two actions related to network stuff can be too much

---

Very good point. I see how it is very network centric with two suggestions. My thought was the same as yours "IoCs can be a very broad term", so what exactly are we recommending a "block" for as those typically pertain to network traffic.

---

Some security solutions allows blocklists of hashes, processes, etc.. It is more like a intentional generic recommendation to cover these actions, but we can work on something more specific if needed

---

If we're going with "IoCs" here, I suggest spelling out the term first:
```suggestion
  - Immediately block the identified indicators of compromise (IoCs).
```